src/containers/Layout/Header.tsx

```diff
@@ -79,8 +79,8 @@ const Header: React.FC<HeaderProps> = ({ onNotificationBadgeClick }) => {
     className="header"
     aria-label="Page Header"
     headerTools={pageToolbar}
-    logo={(<React.Fragment><img src={logo} className="logo" alt="darwinAI" height="300px" width="190px"/>
-    <span className='logo-text'>COVID-Net</span></React.Fragment>)}
+    logo={(<React.Fragment><a onClick={(e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {history.push('/'); e.preventDefault();}} href="/#"><img src={logo} className="logo" alt="darwinAI" height="300px" width="190px"/></a>
+    <a className='logo-text' onClick={(e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {history.push('/'); e.preventDefault();}} href="/#">COVID-Net</a></React.Fragment>)}
```

Comment:
In my personal opinion, I'd not use inline TS code for the onClick events. Leave this as it is for now. However, as a team, we might want to discuss later in the future what would be the best practices for this project such as where inline TS code is acceptable.

Reply:
Understood, I went with the quick inline fix since it was the shortest solution to fix the issue with the anchors. Ideally, we should be using buttons instead of anchors here, as we discussed earlier

Reply:
Resolving the comment for now, something to discuss later on as a team.